src/js/BookReader/Mode1Up.js

```diff
@@ -12,144 +12,219 @@ export class Mode1Up {
   constructor(br, bookModel) {
     this.br = br;
     this.book = bookModel;
+
+    /** @private */
+    this.$documentContainer = $('<div class="BRpageview" />');
```

Comment:
Q: does this get bound to `br.refs`? do we need to?

Reply:
Nope! Only used by Mode1Up, so no need to expose it globally :)